py/tests/algorithms/anomaly.py

```diff
@@ -39,12 +39,12 @@ def testZeroAnomaly(self):
     """
 
     activeCols = SDR(100)
-    predictiveCells = SDR([100, 30])
+    predictiveCols = SDR(100)
```

Comment:
does this file run? I think the pytest files need be named `*_test.py`

Reply:
ups, you are right, needs to be _test.py